code/modules/mob/living/silicon/robot/sprites/_sprite_datum.dm

```diff
@@ -12,6 +12,8 @@
 	var/has_custom_open_sprites = FALSE
 	var/has_vore_belly_sprites = FALSE
 	var/has_vore_belly_resting_sprites = FALSE
+	var/has_sleeper_light_indicator = FALSE //Moved here because there's no reason lights should be limited to just medical borgs. Or redefined every time they ARE used.
```

Comment:
Initially I wanted to get rid of sleeper light indicators entirely. Of borgs that use them, it's only medical ones and even then only SOME of them, not all. I am not against this, but I am questioning how many borgs will actually use it.

Reply:
I was considering making the light indicator more open, and possibly also allow other colors for other modes, but I feel leaving it as an option is a good thing, and making it be handled by snowflake procs just makes it ick and harder to debug (it certainly was when initially trying to make this PR for chomp) I definitely want to make lights vore-belly based, however